state/aws/dynamodb/dynamodb.go

```diff
@@ -25,7 +25,6 @@ import (
 
```

Comment:
We must be calling `authProvider.Close()` in every place it is consumed. We are not doing that.